dot/state/block.go

```diff
@@ -444,6 +444,13 @@ func (bs *BlockState) SetFinalizedHash(hash common.Hash, round, setID uint64) er
 			return err
 		}
 
+		if des, err1 := bs.IsDescendantOf(rem, hash); des {
+			if err1 != nil {
+				logger.Error("pruning block error", "error", err)
+			}
+			continue
+		}
```

Comment:
I think you can ignore the error here, the only times an error would be returned are if `rem` or `hash` don't exist which should never be the case